### PR TITLE
Adding in OWNERS file.

### PR DESCRIPTION
**Pull request by meowfaceman**
The OWNERS file is necessary for prow integration and has been added.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+reviewers:
+  - meowfaceman
+  - clcollins
+  - jeefy
+approvers:
```
Comment (jeefy) [inline OWNERS]: @jeremyeder should probably be in here as well even if he only plans to drive-by lgtm/review.

Comment (meowfaceman) [reply OWNERS]: Good point. Will do!
